sdk/nodejs/runtime/mocks.ts

```diff
@@ -114,13 +167,14 @@ export class MockMonitor {
 
     public async registerResource(req: any, callback: (err: any, innerResponse: any) => void) {
         try {
-            const result = this.mocks.newResource(
-                req.getType(),
-                req.getName(),
-                deserializeProperties(req.getObject()),
-                req.getProvider(),
-                req.getImportid(),
-                req.getCustom());
+            const result = this.mocks.newResource({
+                type: req.getType(),
+                name: req.getName(),
+                inputs: deserializeProperties(req.getProperties()),
```

Comment:
Shoudl this be `deserializeProperties(req.getObject()),`? Does `req.getProperties` exist for register resource?

Reply:
There might be a similar issue with `getId` vs `getImportid`, but not sure.